src/main/resources/project/forms/CreateConfigurationForm.xml

```diff
@@ -50,7 +50,7 @@
     </formElement>
     <formElement>
         <type>credential</type>
-        <label>Active Directory Application ID/Secret:</label>
+        <label>Client ID/Secret Key:</label>
         <property>credential</property>
```

Comment:
Add these two tags also so that the user name and password field labels also get updated accordingly.
        <userNameLabel>Client ID:</userNameLabel>
        <passwordLabel>Secret Key:</passwordLabel>
        <retypePasswordLabel>Retype Secret Key:</retypePasswordLabel>